### ISSUE TITLE
Deprecated class in pandas 2.0

### ISSUE THREAD

**Issue by spiousas** [OxWearables/biobankAccelerometerAnalysis#222]
I freshly installed accelerometer in a new Python Venv and it installed the last version of pandas (2.0). When I tried to run a tutorial I found an error related to a deprecated class in pandas 2.0.

```
from pandas import (to_datetime, Int64Index, DatetimeIndex, Period,
ImportError: cannot import name 'Int64Index' from 'pandas' (/Users/ignacio/opt/anaconda3/lib/python3.8/site-packages/pandas/__init__.py)

```

**Comment by chanshing** [comment]
Thanks for bringing this up. Can you try downgrading it: `pip install pandas==1.2.5` ?

**Comment by spiousas** [comment]
Yes, I solved it by downgrading to `pandas==1.2.4` and it works perfectly, but if you follow the tutorial it installs the last pandas version by default. 

Thanks!

**Comment by chanshing** [comment]
This should be fixed now the newest release. Thanks for spotting it!
